src/utils/resolve-project-base-path.ts

```diff
@@ -1,5 +1,6 @@
 import chalk from 'chalk';
 import { cosmiconfigSync } from 'cosmiconfig';
+import * as jsoncParser from 'jsonc-parser';
```

Comment:
Can you import just what you need?

Reply:
I can import `parse` and `printParseErrorCode` directly, but without renaming (`import { parse, printParseErrorCode } from 'json-parser'` vs. `import { parse as parseJson, printParseErrorCode as printJsonParseErrorCode } from 'json-parser'`) it will be very confusing. But if we are going to rename the imports, in my opinion it is easier to use the namespace import.

Reply:
Thinking about it again I think that this should be moved to a dedicated file that exports this "json loader".
There you won't have any confusion since this file will only host this json loader.

Reply:
looks good, left a rename suggestion :)